test/hooks.test.js

```diff
@@ -1269,7 +1270,15 @@ test('clear payload', t => {
 })
 
 test('onSend hook throws', t => {
-  t.plan(9)
+  t.plan(11)
+  const Fastify = proxyquire('..', {
+    './lib/schemas.js': {
+      getSchemaSerializer: (param1, param2, param3) => {
+        t.equal(param3, 'application/json; charset=utf-8', 'param3 should be "application/json; charset=utf-8"')
+        t.end()
```

Comment:
Do not end the test yourself.
```suggestion
```

Reply:
> Do not end the test yourself.

Done. I would appreciate it if you could share it with me why?

Reply:
t.plan will assert if the assertion number is match what it's needed and end accordingly.

When you forcefully end the test. All assertion check will not run at that point and end no matter it success or fail.

Reply:
> t.plan will assert if the assertion number is match what it's needed and end accordingly.
> 
> When you forcefully end the test. All assertion check will not run at that point and end no matter it success or fail.

Thank you for sharing your knowledge with me and for your time @climba03003